config/config.h.in

```diff
@@ -4,7 +4,10 @@
 #cmakedefine DEBUG
 
 #define SIZEOF_INT @SIZEOF_INT@
+#define SIZEOF_INTMAX_T @SIZEOF_INTMAX_T@
 #define SIZEOF_LONG @SIZEOF_LONG@
+#define SIZEOF_LONG_LONG @SIZEOF_LONG_LONG@
+#define SIZEOF_SIZE_T @SIZEOF_SIZE_T@
```

Comment:
wonder if it makes sense to update these to `#cmakedefine` ?

Reply:
yeah this works as well, I opted to keep the longer version hoping to avoid any potential problems, or do you think it's fine either way?

refs: 
- https://cmake.org/cmake/help/v3.10/module/CheckTypeSize.html

external example:
- https://opensource.apple.com/source/curl/curl-121.100.3/curl/lib/curl_config.h.cmake